client/components/ReviewRating/ReviewList/ReviewRating.jsx

```diff
@@ -37,13 +54,16 @@ const ReviewRating = ({ id, rating }) => {
     setIsClickAdd(false);
   };
   return (
+
```

Comment:
Did you mean to include an empty line here? It looks strange. 🤔

Reply:
no, that is an extra line and I removed it.